Makefile

```diff
@@ -158,4 +158,32 @@ local-uninstall-chrome:
 
 local-uninstall: local-uninstall-firefox local-uninstall-chrome
 
-.PHONY: clean release local-install-firefox local-install-chrome local-install install local-uninstall-firefox local-uninstall-chrome local-uninstall uninstall
+PV = $(shell echo $(RELEASE_TAG) | sed -e s:^v::)
+PN = linux-entra-sso
+DESCRIPTION = Native app for Entra ID SSO via Microsoft Identity Broker
```

Comment:
This is not just the native part, but also includes the download-stub for Chrome. Either we remove that stub, or we change the description to better reflect what the package contains.